### ISSUE TITLE
Using jsonb instead of hstore

### ISSUE THREAD

**Issue by posikifi** [qwat/pg-history-viewer#27]
Hi!

This is maybe suggestion and/or question: is it possible to change storage of audit-tables from hstore to jsonb? 

Pros
- jsonb is part of core PostgreSQL, hstore still needs extension
- jsonb is newer has more functionalities (?)

Cons
- It work, no need to change
- plugin need modification

Peformance: there is a lot of discussion about this, but I haven't seen permanent answer. It depends.

P

**Comment by haubourg** [comment]
I agree and the upstream pg_audit has been abandoned and switched to jsonb logic. 
To me it is essentially a matter of funding. I could be interested in supporting a part of it if efforts can be merged. This tool can really be helpful in many applications!

cc @vpicavet  @ponceta please gather other potential funders / contributors here if you hear of some interest.

**Comment by posikifi** [comment]
I also suggest that we will include/merge https://gitlab.com/Oslandia/audit_trigger into this project.

Another suggestion is that we rename audit_trigger to history_trigger (or something). Reasoning: pgAudit is more auditing (including select etc.) and it has totally different scope.

I can use some time for this and is external funding is needed, let's see what we can do.

**Comment by lbartoletti** [comment]
@posikifi There is (was) a PR for this purpose https://gitlab.com/Oslandia/audit_trigger/-/merge_requests/2/diffs

It takes a little time to update and test, but I know this was discussed a few years ago.

**Comment by ponceta** [comment]
https://www.cybertec-postgresql.com/en/tracking-changes-in-postgresql/ Here is some interesting discussion in the comments.
